index.html

```diff
@@ -11,7 +11,7 @@
             window.onload = function () {
                 // Begin Swagger UI call region
                 const ui = SwaggerUIBundle({
-                    url: "./builds/OpenSearch.latest.yaml", //Location of Open API spec in the repo
+                    url: "https://github.com/opensearch-project/opensearch-api-specification/archive/refs/tags/OpenSearch.latest.yaml",
```

Comment:
This URL doesn't look correct to me, do you have an example of a functioning URL for your fork?

Reply:
Yep, you're right. Updated to match the release URL. Here's one that works from my branch https://github.com/dblock/opensearch-api-specification/releases/download/latest-draft-release/OpenSearch.latest.yaml

Btw, I don't love these names "OpenSearch.latest.yaml", but we can rename the files in another PR.